src/chart/generateCategoricalChart.tsx

```diff
@@ -2184,6 +2185,18 @@ export const generateCategoricalChart = ({
             }),
           ];
         }
+      } else if (hasActive) {
+        /**
+         * We hit this block if consumer uses a Tooltip without XAxis and/or YAxis.
+         * In which case, this.state.activeTooltipIndex never gets set
+         * because the mouse events that trigger that value getting set never get trigged without the axis components.
+         *
+         * See FunnelChart story for an example use case
```

Comment:
I suggest to remove this reference, because (a) it is not specific enough (there might be many stories), and (b) we will most certainly forget to remove this if we ever remove the story. 
```suggestion
         * An example usage case would be a FunnelChart
```